### DISCUSSION TITLE
Switching hacktribe/original OS back and forth

### DISCUSSION THREAD

**Discussion by fakrij** [bangcorrupt/hacktribe#231] [Q&A] [answered]
TL;DR: I switched to hacktribe but encountered problems with my projects in the device and didnt wanted to lose so much work. I switched back to original OS but now cant switch to hacktribe again but im getting "Error, invalid file".

More info:
Hi, I recently upgraded to hacktribe. I thought there will be some extra work to migrate samples (easy one) and also patterns I had in the device. It turned out though, that the oscilator numbers were not remembered and instead of that it just deleted them, leaving empty instruments in my patterns. I tried to read the patterns in my pc using some max4live manager for E2 (blue/grey) but it didnt work unfortunatelly.

So i decided to switch the OS back to original 2.02. It worked like charm, I did my notes on what has to be done but now I cant switch back to hacktribe with "Error, invalid file". Im sure Im using the SYSTEM.VSB I used for the first time hacktribe upgrade... Any clues on what can I do now?

Thank you
j

**Comment by bangcorrupt** [comment, marked as answer]
That error message usually means the file has the wrong device ID in the header.  If you are installing Hacktribe from synth firmware you will need to edit the header to look like the synth firmware. 

    python e2-header.py hacked-SYSTEM.VSB synth

 The synth only see files under the `KORG/electribe` directory, check the hacked SYSTEM.VSB is at the correct location: `KORG/electribe/System/SYSTEM.VSB`.

EDIT: Swapped `sampler` with `synth` above.

**Reply by fakrij** [reply]
Oh, my bad, sorry about that but I didnt mention Im patching E2S. So I had E2S with 2.02 original >>> hacktribe >>> 2.02 E2S original >>> file corrupted.
